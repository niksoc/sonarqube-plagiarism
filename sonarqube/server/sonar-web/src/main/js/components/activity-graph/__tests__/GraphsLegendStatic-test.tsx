/*
 * SonarQube
 * Copyright (C) 2009-2020 SonarSource SA
 * mailto:info AT sonarsource DOT com
 *
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the GNU Lesser General Public
 * License as published by the Free Software Foundation; either
 * version 3 of the License, or (at your option) any later version.
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with this program; if not, write to the Free Software Foundation,
 * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 */
import { shallow } from 'enzyme';
import * as React from 'react';
import GraphsLegendStatic from '../GraphsLegendStatic';

const SERIES = [
  { name: 'bugs', translatedName: 'Bugs', data: [] },
  { name: 'code_smells', translatedName: 'Code Smells', data: [] }
];

it('should render correctly the list of series', () => {
  expect(shallow(<GraphsLegendStatic series={SERIES} />)).toMatchSnapshot();
});
