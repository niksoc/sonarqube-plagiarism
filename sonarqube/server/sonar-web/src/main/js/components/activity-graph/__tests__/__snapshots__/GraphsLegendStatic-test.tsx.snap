// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly the list of series 1`] = `
<div
  className="activity-graph-legends"
>
  <GraphsLegendItem
    className="big-spacer-left big-spacer-right"
    index={0}
    key="bugs"
    metric="bugs"
    name="Bugs"
  />
  <GraphsLegendItem
    className="big-spacer-left big-spacer-right"
    index={1}
    key="code_smells"
    metric="code_smells"
    name="Code Smells"
  />
</div>
`;
